core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/IntelliJVersion.scala

```diff
@@ -31,7 +31,7 @@ final case class IntelliJVersion(build: String, release: Option[String]) {
 object IntelliJVersion {
   implicit val configConvert: ConfigConvert[IntelliJVersion] = deriveConvert[IntelliJVersion]
 
-  val Latest = release("2021.2.1", "212.5080.55")
+  val Latest: IntelliJVersion = release("2021.2.1", "212.5080.55")
```

Comment:
should it be read from config now?

Reply:
Yes, but it is also used by the default values of the `IntelliJFixture`. We can remove this in changes for this issue: https://github.com/VirtusLab/ide-probe/issues/253